var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -286,20 +311,8 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage, PythonExtension):
     # depends_on('android-sdk', when='+android')
 
     with when("+rocm"):
-        depends_on("hip")
-        depends_on("rocrand")
-        depends_on("rocblas")
-        depends_on("rocfft")
-        depends_on("hipfft")
-        depends_on("rccl", when="+nccl")
-        depends_on("hipsparse")
-        depends_on("hipcub")
-        depends_on("rocsolver")
-        depends_on("rocprim")
-        depends_on("miopen-hip")
-        depends_on("llvm-amdgpu")
-        depends_on("hsa-rocr-dev")
-        depends_on("rocminfo")
+        for pkg_dep in rocm_dependencies:
+            depends_on(pkg_dep)
```

Comment:
Is there any information about the minimum supported versions for these deps?

Reply:
restricted 2.14 to ROCm 6.0 and higher